implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -288,6 +288,19 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
     @ConfigProperty(name = "smallrye.jwt.verify.algorithm", defaultValue = "RS256")
     private Optional<SignatureAlgorithm> signatureAlgorithm;
 
+    /**
+     * Verify the certificate thumbprint.
+     * If this property is enabled then a signed token must contain either 'x5t' or 'x5t#256' X509Certificate thumbprint
+     * headers.
+     * Verification keys can only be in JWK or PEM Certificate key formats in this case.
+     * JWK keys must have a 'x5c' (Base64-encoded X509Certificate) property set.
+     * Note that 'smallrye.jwt.token.kid' property will have no effect as 'x5t' and 'x5t#S256'
+     * are the key identifiers when this property is set.
+     */
+    @Inject
+    @ConfigProperty(name = "smallrye.jwt.verify.certificateThumbprint", defaultValue = "false")
+    private boolean verifyCertificateThumbprint;
```

Comment:
Does this need a line in `getOptionalContextInfo` to pass to the injected value to `JWTAuthContextInfo`?

Reply:
@MikeEdgar sorry, bad omission, in `DefaultJWTParserTest' the property is set in the test, so I added 2 more tests starting from [here](https://github.com/smallrye/smallrye-jwt/pull/302/files#diff-6547c7bfe75cb1091761931d682eccd4R124), have a look please.
There should be an easier way to bootstrap `JWTAuthContextInfoProvider` (may be just set all those properties as public, etc) but for now I'm keeping adding the helper methods